src/generic_ui/polymer/invite-user.html

```diff
@@ -221,27 +215,15 @@
           </div>
 
           <div id='allNetworkButtons'>
-
-            <div id='cloudInstall' hidden?='{{ model.cloudProviderNames.length < 1 }}' on-tap='{{ cloudInstall }}'>
-              <div class='content'>
-                <img src='../../icons/cloud/connect-cloud.png'>
-                <h1>{{ 'CLOUD_INSTALL_INVITE_SCREEN_TITLE' | $$ }}</h1>
-                <p>{{ 'CLOUD_INSTALL_INVITE_SCREEN_MESSAGE' | $$ }}</p>
-              </div>
-              <div class='chevronWrapper'>
-                <core-icon icon="chevron-right"></core-icon>
-              </div>
-            </div> <!-- end of cloud -->
-
-            <div class='categoryHeader'>{{ "CHOOSE_A_SOCIAL_NETWORK" | $$ }}</div>
-
             <template repeat='{{ n in model.networkNames }}' vertical layout>
-              <div hidden?='{{ n == "Cloud" }}' on-tap='{{ networkTapped }}' class='networkLoginButton' data-network='{{n}}'>
+              <div on-tap='{{networkTapped }}' class='networkLoginButton' data-network='{{n}}'>
                 <core-icon icon='uproxy-icons:{{n}}'></core-icon>
                 <div class='networkName'>{{getNetworkDisplayName(n)}}</div>
+                <!--
                 <span class='experimental paleText' hidden?='{{!isExperimentalNetwork(n)}}'>
                   (Experimental)
                 </span>
+                -->
```

Comment:
Note: the new design specs have removed the "(experimental)" labels from the UI.